libpromises/cf3lex.l

```diff
@@ -217,6 +218,45 @@ promise_type   [a-zA-Z_]+:
                         }
                       }
 
+{macro_if_between_versions} {
+                        if ( P.line_pos != 1 )
```

Comment:
Why inconsistent spacing with the rest of the function?

Reply:
I hate this indentation style. I want to fix it later. With longer rule names, the code just goes off the side of the editor.